webpack.multi.js

```diff
@@ -0,0 +1,181 @@
+const path = require('path');
+const webpack = require('webpack');
+const config = require('./config');
+
+// plugins
+const ExcludeAssetsPlugin = require('webpack-exclude-assets-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const WebpackRTLPlugin = require('webpack-rtl-plugin');
+
+const plugins = [];
+
+const appRoot = path.resolve('./');
+const jsSource = `./${config.sourcePath}/${config.jsDirectory}`;
+const cssSource = `./${config.sourcePath}/${config.cssDirectory}`;
+
+const jsEntries = [
+  { main: [`${jsSource}/main.js`] },
+  { raven: [`${jsSource}/raven.js`] },
+  { styleguide: [`${jsSource}/styleguide/styleguide.jsx`] },
+  { survey: [`${jsSource}/surveys/survey.js`] },
+  { survey_admin: [`${jsSource}/surveys/survey-admin.js`] },
+  { survey_results: [`${jsSource}/surveys/survey-results.jsx`] },
+  { campaigns: [`${jsSource}/campaigns.js`] },
+  { charts: [`${jsSource}/charts.js`] },
+  { tinymce: [`${jsSource}/tinymce.js`] },
+  { embed_course_stats: [`${jsSource}/embed_course_stats.js`] },
+];
+
+const cssEntries = [
+  { main: [`${cssSource}/main.styl`] },
+  { styleguide: [`${cssSource}/styleguide.styl`] },
+  { surveys: [`${cssSource}/surveys.styl`] },
+  { training: [`${cssSource}/training.styl`] },
+];
+
+const cssConfig = env => ({
```

Comment:
This file has a lot going on, in terms of both composing the separate sets of entries for parallel processing and also configuring sets of plugins for CSS vs JS.

Can you refactor this to separate the concerns more fully? Ideally, there would be one file that we can look at to understand the JS configuration, another for the CSS configuration, and this file would just be the glue to make them work with parallel-webpack.